ui/components/ui/icon-button/icon-button.js

```diff
@@ -23,9 +23,15 @@ export default function IconButton(props) {
 
   return (
     <button
-      className={classNames('icon-button', className, {
-        'icon-button--disabled': disabled,
-      })}
+      className={classnames(
+        'icon-button',
+        'w-16',
+        {
+          'icon-button--disabled': disabled,
+          'opacity-30 cursor-auto': disabled,
+        },
+        className,
+      )}
```

Comment:
Refactored to use Tailwind classnames. Many of the original styles for this class weren’t actually needed (e.g., all flexbox-related classnames). We're retaining the `icon-button` and `icon-button--disabled` classnames to avoid breaking existing tests. If this were a separate PR, we could consider removing these classnames entirely in favor of data test IDs or other selector strategies.